src/tape/ensemble_readers.py

```diff
@@ -10,7 +10,15 @@
 from tape.utils import ColumnMapper
 
 
-def read_ensemble(dirpath, additional_frames=True, column_mapper=None, dask_client=True, **kwargs):
+def read_ensemble(
+    dirpath,
+    additional_frames=True,
+    column_mapper=None,
+    dask_client=True,
```

Comment:
Nit: move dask_client to reflect Parameter order in the doc string (or vice versa)